admin/build.sbt

```diff
@@ -0,0 +1,46 @@
+lazy val piezoAdmin = (project in file(".")).enablePlugins(PlayScala)
+
+bashScriptExtraDefines ++= Seq(
+  "mkdir -m 664 /var/run/piezo-admin/",
+  s"chown ${(daemonUser in Linux).value} /var/run/piezo/admin",
```

Comment:
you used /var/run/piezo-admin before and here you use /var/run/piezo/admin (FWIW I like the second better)